qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -816,6 +821,128 @@ def test_issue_cephfs_shell_cmd_at_invocation(self):
                    "expected_output -\n{}\ndu_output -\n{}\n".format(
                    dirname, output)
 
+    def test_every_shell_cmd_at_invocation(self):
+        """
+        Runs every test in this suite with
+        ``cephfs-shell -c ceph.conf <cephfs-shell-command>``
+        instead of ``cephfs-shell -c ceph.conf -- <cephfs-shell-command>``
```

Comment:
I don't really feel it's necessary to re-test **every** CephFS shell test with this pattern. Why is it not sufficient to have a specific test that does:

`cephfs-shell -c ceph.conf mkdir foo`

?

Reply:
The idea was to improve test coverage to test that if everything that can be done with interactive shell can also be done with the non-interactive shell.

I think we can keep with test but we may move it to another file or disable it and run it when we want to test cephfs-shell exhaustively (i.e. not run it while testing PRs).

Reply:
I agree with @batrick. We need not retest every test. @rishabh-d-dave  you will disable this test before merging ?

Reply:
Yes.

Reply:
This test appears to still be in the PR even though you're asking for final merge @rishabh-d-dave, please explain.

Reply:
What I meant in our last meeting was if it was possible to merge *if* those couple of failures were classified as not relevant to the PR. If I would've gotten a yes from you back then, I would've proceed to make this change (disabling the test) and I would've requested you to comment that on the PR so that Varsha would know when she would look at the results from integration testing.

Reply:
Anyways, I'll go ahead label this PR DNM until we decide on those results and I make the this change.

Reply:
@varshar16 
> I agree with @batrick. We need not retest every test. @rishabh-d-dave you will disable this test before merging ?

Done.